### PR TITLE
feat: python 3.13 support

### PR DESCRIPTION
**Pull request by bdraco**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
+      - uses: actions/setup-python@v4
```
Comment (dlech) [inline .github/workflows/ci.yml:18]: why not v5?

Comment (bdraco) [inline .github/workflows/ci.yml:18]:
```suggestion
      - uses: actions/setup-python@v5
```

Comment (bdraco) [reply .github/workflows/ci.yml]: copied from another template and didn't update.  updated now

**Comment by bdraco** [comment]
Probably ok to merge but I need to babysit the wheel builds and its too late to do that since I need to get up early for a flight. Will have to do it later this week when I have more time.

**Pull request merged**